dist/api_data.py

```diff
@@ -1471,13 +1471,15 @@ def __cmp__(self, other):
     Config('get', 'all_committed', r'''
         specify which timestamp to query: \c all_committed returns the largest
         timestamp such that all timestamps up to that value have committed,
+        \c last_checkpoint returns the timestamp of the last stable checkpoint,
         \c oldest returns the most recent \c oldest_timestamp set with
         WT_CONNECTION::set_timestamp, \c oldest_reader returns the
         minimum of the read timestamps of all active readers \c pinned returns
-        the minimum of the\c oldest_timestamp and the read timestamps of all
-        active readers, and \c stable returns the most recent
-        \c stable_timestamp set with WT_CONNECTION::set_timestamp. See
-        @ref transaction_timestamps''',
+        the minimum of the \c oldest_timestamp and the read timestamps of all
+        active readers, \c recovery returns the timestamp of the last stable
+        checkpoint taken prior to an unclean shutdown and \c stable returns the
```

Comment:
I don't think it's just unclean shutdowns? Clean shutdowns create a checkpoint with a timestamp as well.